packages/desktop-client/src/components/budget/report/BudgetSummary.tsx

```diff
@@ -280,7 +279,11 @@ function Saved({ projected, style }: SavedProps) {
           className={`${css([
             {
               fontSize: 25,
-              color: projected ? colors.y3 : isNegative ? colors.r4 : colors.p5,
+              color: projected
+                ? theme.warningText
+                : isNegative
+                ? theme.errorText
+                : theme.pageTextPositive,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** significantly different color used